packages/cozy-harvest-lib/src/datacards/FileDataCard.jsx

```diff
@@ -181,8 +184,30 @@ const makeQueryFromProps = ({ accountId }) => ({
   fetchPolicy: CozyClient.fetchPolicies.olderThan(30 & 1000)
 })
 
-const FileDataCard = ({ filesCol, konnector }) => {
-  const { data: files } = filesCol
+const FileDataCard = ({ filesCol, konnector, accountId }) => {
+  const client = useClient()
+  const { data: files, fetch } = filesCol
+
+  const debouncedFetch = useMemo(() => throttle(fetch, 300), [fetch])
+  const onFileCreation = useCallback(
+    file => {
+      if (get(file, 'cozyMetadata.sourceAccount') === accountId) {
+        debouncedFetch()
+      }
+    },
+    [accountId, debouncedFetch]
+  )
+  useRealtime(
+    client,
+    {
+      'io.cozy.files': {
+        created: onFileCreation,
+        updated: onFileCreation
+      }
+    },
+    [onFileCreation]
+  )
```

Comment:
I think we don't have to do this. We can use: 

`<RealTimeQueries doctype='io.cozy.files' />`. And since this component will dispatch `CozyClient's Mutation`, its store will be updated, and the result of the query will be updated too, no?

Reply:
Nice, I was able to replace my code by RealTimeQueries easily !